apps/ocpp/store/pending_calls.py

```
@@ -0,0 +1,424 @@
+                action_label = action or str(metadata.get("action") or "Call")
+                entry_label = f"{action_label} request timed out"
+            metadata["timeout_notice_sent"] = True
+        if target_log and entry_label:
+            add_log(target_log, entry_label, log_type=log_type)
+
+    future: concurrent.futures.Future[asyncio.TimerHandle] = concurrent.futures.Future()
+
+    def _schedule_timer() -> None:
+        try:
+            handle = loop.call_later(timeout, _notify)
+        except Exception as exc:  # pragma: no cover - defensive
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)